Monika-After-Story/MonikaModDev
# Changed format of sprite gift persistents so giftname is less important

basically makes it possible to change giftname of sprites that we release custom dialogue for

**NOTE:** jmo giftnames are now:
* `cherryhairclip`
* `hearthairclip`
* `musicnotehairclip`

**YOU MUST CLEAR THE FOLLOWING PERSISTENTS BEFORE TESTING:**
* `_mas_filereacts_sprite_reacted`
* `_mas_sprites_json_gifted_sprites`
* `_mas_filereacts_sprite_gifts`

# Testing
* verify that sprite gifts still work
* verify that changing the jsons of jmo sprites and using different giftnames still work
